<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusWidget</class>
 <widget class="QWidget" name="StatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>17</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>SMD positon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSmdPosition">
     <property name="text">
      <string>##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbStatus">
     <property name="text">
      <string>##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>CNT status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbCntStatus">
     <property name="text">
      <string>##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>SMD status:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSmdStatus">
     <property name="text">
      <string>##</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CNT error count:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbCntErrorCount">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>SMD error count:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSmdErrorCount">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Secnod read:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbSecondRead">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
